writeJSONToFile.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"crypto/sha256"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"time"
+)
+
+func writeJSONToFile(resp *http.Response) error {
+	// Writing the API Data to a File
+	shaHash := sha256.New()
```

Comment:
I believe it is more pragmatic to use the `uuid` package instead.

Reply:
Done